wework/src/styles/globals.css

```
@@ -156,6 +156,143 @@ button:disabled {
+  }
+
+  .file-change-rolling-value.is-rolling {
+    animation: file-change-number-roll 320ms cubic-bezier(0.2, 0.82, 0.18, 1);
+  }
+
+  .file-change-delta-badge {
+    position: absolute;
+    inset-inline-start: 100%;
+    inset-block-start: -0.8em;
+    margin-inline-start: 2px;
+    color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)